src/dbnode/generated/thrift/rpc.thrift

```diff
@@ -468,6 +468,34 @@ struct QueryRequest {
 	7: optional TimeType rangeType = TimeType.UNIX_SECONDS
 	8: optional TimeType resultTimeType = TimeType.UNIX_SECONDS
 	9: optional binary source
+	// Additional options for the Cluster service.
+	10: optional ClusterQueryOptions clusterOptions;
```

Comment:
nit: Don’t need semi colon at end of line (for consistency with other lines in this file per field)